src/server.ts

```diff
@@ -49,9 +74,15 @@ const parseDomainSingle = function(domainTargetPort: string){
 }
 
 if (perDomainRaw != undefined){
-  perDomainRaw.split(',').forEach((s) => parseDomainSingle(s))
+  for(let domain in perDomainRaw) parseDomainSingle(perDomainRaw[domain])
 }
 
+// no-proxy function
+const toggleReproxy = (domain: string) => {
+  Object.keys(perDomainRaw).includes(domain)
+    ? disableReproxy = true
+    : disableReproxy = false
+}
```

Comment:
@arunaruljothi 
This toggles the `disableProxy` value, feel free to include it when you need it in future